assemble/bin/accumulo-service

```diff
@@ -91,27 +91,78 @@ function start_service() {
   fi
 }
 
-function stop_service() {
+function control_process() {
+  kill_code=${1}
```

Comment:
Code looks cleaner without the unnecessary use of curlies. Also, local variable FTW

```suggestion
  local kill_code=$1
```